### DISCUSSION TITLE
--dns-host does not create any entry in /etc/hosts

### DISCUSSION THREAD

**Discussion by obfischer** [abiosoft/colima#600] [Q&A]
The option `--dns-host x.domain.y=1.2.3.4` should create according to my understanding an entry in `/etc/hosts`. In Colima 0.5.2 this does not work.

```
colima delete -f
colima sart --dns-host x.domain.y=1.2.3.4
colima ssh -- cat /etc/hosts
```

Currently I use the following command to create the missing entry:

```
echo -e "\n1.2.3.4\tx.domain.y" | colima ssh --  sudo tee -a /etc/hosts
```

Is this a bug or did I miss something?

**Comment by abiosoft** [comment]
Actually, it does not populate the `/etc/hosts` and only uses the internal resolver. 

If you do a `nslookup` for `x.domain.y`, it would respond with `1.2.3.4` accordingly.

> It does not configure the /etc/hosts files of any machine or container.

https://github.com/abiosoft/colima/blob/d2b24d218dfc987f213edf7e655420e14f000153/embedded/defaults/colima.yaml#L64-L73
